apps/web/playwright/fixtures/embeds.ts

```diff
@@ -43,14 +48,18 @@ export const createEmbedsFixture = (page: Page) => {
               // eslint-disable-next-line @typescript-eslint/ban-ts-comment
               //@ts-ignore
               api = window.Cal.ns[calNamespace];
+              console.log("Using api from namespace-", { calNamespace, api });
             }
-            console.log("PlaywrightTest:", `Adding listener for __iframeReady on namespace:${calNamespace}`);
             if (!api) {
-              throw new Error(`namespace "${calNamespace}" not found`);
+              console.log(`namespace "${calNamespace}" not found yet - Trying again`);
+              setTimeout(tryAddingListener, 500);
```

Comment:
Handle the case when namespace is created after a CTA is clicked and not on page load.